src/utils/loggers.js

```diff
@@ -42,7 +43,34 @@ export class WinstonConsoleLogger extends Logger {
     this.logger.info(message);
   }
 
-  error(message) {
-    this.logger.error(message);
+  error(err) {
+    this.logger.error(err.stack ? err.stack : err);
+  }
+}
+
+export class WinstonExpressLogger extends Logger {
+  constructor() {
+    super();
+    const logFormat = format.printf((info) => info.message);
+    this.logger = createLogger({
+      format: logFormat,
+      transports: [
+        new transports.File({
```

Comment:
Again, stdout should be the default here because we run in Docker. Docker takes the stdout of the process it's running and optionally ships those logs somewhere, so we don't have to worry about saving the logs to a file :)

Reply:
👍